### DISCUSSION TITLE
Add macroscopic maxwell solver feature in PICMI WarpX

### DISCUSSION THREAD

**Discussion by GivenChen** [BLAST-WarpX/warpx#4960] [Ideas]
Hi, I am using PICMI to run WarpX since it offers great flexibility in controlling the particle and field data when running the simulation. However, I found that PICMI doesn't have the feature of using the macroscopic Maxwell solver. I am wondering if it is possible that the macroscopic Maxwell solver option can be added to Python PICMI WarpX?

**Comment by RemiLehe** [comment]
That's a good point, thanks. We will add it in PICMI.

**Reply by GivenChen** [reply]
Thank you~
